### DISCUSSION TITLE
feature request: export to DLC / DLC format

### DISCUSSION THREAD

**Discussion by WeissShahaf** [talmolab/sleap#1419] [Ideas]
Dear Devs,
1) Can it be possible to export infferred pose coordinates as a csv in the DLC format of output? 
Use case: let's say i want to use SLEAP for pose estimation, but take the output to a pipeline that uses DLC CSVs as input.

2) Is it possible to export labeled data back to a DLC folder? 
Use case: using SLEAP's great labeling GUI, and saving it back to a DLC folder to do the training there.

Thanks,
Shahaf

**Comment by mollywestbrook** [comment]
Hi Shahaf! Community member here, but awhile ago someone posted a great google colab notebook for this exact purpose under discussion [1089](https://github.com/talmolab/sleap/discussions/1089). Hope this helps!

**Reply by WeissShahaf** [reply]
thanks!
